src/python/modules/ensembl/compara/citest/tabletest.py

```diff
@@ -0,0 +1,398 @@
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the
+# EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+ Module describing the class TableTest
+
+"""
+
+
+#from sqlalchemy import create_engine
+from sqlalchemy.sql import text
+from ensembl.compara.citest.test import Test
```

Comment:
If you need to setup PYTHONPATH for the imports to work, you should document that somewhere, update the environment module and the Travis scripts